src/locales/zh.json

```diff
@@ -226,6 +226,7 @@
       "args": "Args: ",
       "hash_type": "Hash Type: ",
       "assets": "资产",
+      "overview": "概览",
```

Comment:
`address.assets` is no longer used and should be removed to try to avoid more and more useless code.